### DISCUSSION TITLE
In vLLM v0.5.5, does --num-scheduler-steps not work with --enable-lora?

### DISCUSSION THREAD

**Discussion by ashgold** [vllm-project/vllm#8093] [Q&A]
In vLLM v0.5.5, does --num-scheduler-steps not work with --enable-lora?

I'm serving lora adapters via --enable-lora.
But when I set --num-scheduler-steps=8 and reason with adapter, the answer generated is the answer from the base model.

In vLLM v0.5.5, --num-scheduler-steps cannot be mixed with --enable-lora?

**Comment by comaniac** [comment]
It's not supported yet and being tracked by #7528.

**Reply by ashgold** [reply]
Thanks for your quick reply!
